### ISSUE TITLE
Character set 'utf8' unsupported

### ISSUE THREAD

**Issue by YEUAM** [karlrink/db-api#2]
hello ,  nice Tool

But i got an error 
"Character set 'utf8' unsupported"

any suggestions , how to solve that ?

```bash
ubuntu@yaskdev:~
$ curl http://127.0.0.1:8980/
{
  "status": 200,
  "message": "OK",
  "version": "1.0.5"
}

ubuntu@yaskdev:~
$ curl --user dbuser:dbpass http://127.0.0.1:8980/api
{
  "status": 512,
  "errorType": "ProgrammingError",
  "errorMessage": "Character set 'utf8' unsupported"
}
ubuntu@yaskdev:~

````

**Comment by YEUAM** [comment]
closed .. works  with the docker container

**Comment by rink-us** [comment]
utf8 Error may be related to the mysql.connector version.  I’m still looking myself.

On Oct 18, 2022, at 6:10 AM, Andreas Meier ***@***.***> wrote:

﻿


Closed #2 as completed.

**Comment by YEUAM** [comment]
rebuild the  docker container with  :

Dockerfile
```Dockerfile 
FROM python:3.8-slim-buster
....
```
and 

requirements.txt
``` requirements.txt
flask
flask-cors
mysql-connector-python==8.0.29
# gunicorn
```

That seems to "fix"  the Problem , by  downgrading the  "mysql-connector-python" modul to  v8.0.29
and the Docker image  is then less the 250 MB ,   instead of > 1GB   

found some info :
https://stackoverflow.com/questions/73244027/character-set-utf8-unsupported-in-python-mysql-connector
